src/viewer.js

```diff
@@ -140,16 +140,17 @@ class Viewer {
   
   /**
    * @param {boolean} isLastBack true if back button was hit and viewer should hide.
-   * @param {number} opt_poppedAmpHistoryIndex The AMP History Index that was just popped.
+   * @param {boolean} isAMP true if going to AMP document.
+   * @param {number|undefined} opt_poppedAmpHistoryIndex The AMP History Index that was just popped.
```

Comment:
I think this is still a bit unclear. When you pop, the state in the event will be that of the incoming state, not the previous state. I think it's confusing what is meant by "just popped".